app/templates/components/order-card.hbs

```diff
@@ -12,6 +12,11 @@
         <div class="ui {{order-color order.status}} large label">
           {{order.status}}
         </div>
+        {{#if isUserCheckedIn}}
+          <div class="ui green large label">
+            {{t 'Checked In'}}
+          </div>
+        {{/if}}
```

Comment:
@shreyanshdwivedi an alternate "not checked in" label would be lovely too!

Reply:
@uds5501 I've added `Not Checked In` also. Please refer to screenshot attached to PR description
Will work on fixing Travis failure now